client/less/main.less

```diff
@@ -1179,4 +1179,8 @@ and (max-width : 400px)  {
 @import "jobs.less";
 @import "challenge.less";
 @import "toastr.less";
-@import "map.less";
\ No newline at end of file
+@import "map.less";
+
+ol, li {
```

Comment:
This will render any and all `ol` and `li` on all pages that use this including the challenges.